src/jdk.compiler/share/classes/com/sun/tools/javac/code/Preview.java

```diff
@@ -115,7 +119,22 @@ private Map<Integer, Source> initMajorVersionToSourceMap() {
             }
         }
         return majorVersionToSource;
-   }
+    }
+
+    /**
+     * Returns true if {@code s} is participating in the preview of {@code previewSymbol}, and
+     * therefore no warnings or errors will be produced.
+     *
+     * @param s the symbol depending on the preview symbol
+     * @param previewSymbol the preview symbol marked with @Preview
+     * @return true if {@code s} is participating in the preview of {@code previewSymbol}
+     */
+    public boolean isPreviewParticipating(Symbol s, Symbol previewSymbol) {
```

Comment:
Some feedback from a colleague:
```suggestion
    /**
     * Returns true if {@code s} is deemed to participate in the preview of {@code previewSymbol}, and
     * therefore no warnings or errors will be produced.
     *
     * @param s the symbol depending on the preview symbol
     * @param previewSymbol the preview symbol marked with @Preview
     * @return true if {@code s} is participating in the preview of {@code previewSymbol}
     */
    public boolean participatesInPreview(Symbol s, Symbol previewSymbol) {
```

Reply:
Merged.